CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,26 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    /// Override this property if you need to set a specific font
+    /// Defaults to 16 without an icon and 10 with
+    public var titleFont: UIFont? = nil {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var translucentBackground: Bool = false {
```

Comment:
No, I'm thinking that this property is what indicates whether there is a background blur. Maybe rename it to make it clearer if that is not the case. 

I was wanting you to change it to be true, and change the behavior so that it didn't remove the blur.